lib/require-from-worker.js

```diff
@@ -1,43 +1,18 @@
 'use strict';
 
-const debugGenerator = require('heimdalljs-logger');
-const hashForDep = require('hash-for-dep');
 const AstPlugins = require('./ast-plugins');
-
-const _logger = debugGenerator('ember-cli-htmlbars-inline-precompile');
+let astPlugin = {};
 
 module.exports = {
-  build(options) {
-    let pluginInfo = this.setupDependentPlugins(options.parallelConfig);
-
-    return AstPlugins.setup(pluginInfo, {
-      templateCompilerPath: options.templateCompilerPath,
-      useCache: true,
-    });
-  },
-
-  setupDependentPlugins(parallelConfig) {
-    const plugins = [];
-    const cacheKeys = [];
-
-    parallelConfig.forEach((config) => {
-      const plugin = require(config.requireFile);
-      const buildInfo = plugin[config.buildUsing](config.params);
-
-      plugins.push(buildInfo.plugin);
-
-      if (typeof buildInfo.baseDir === 'function') {
-        const pluginHashForDep = hashForDep(buildInfo.baseDir());
-        cacheKeys.push(pluginHashForDep);
-      } else {
-        _logger.debug('ember-cli-htmlbars-inline-precompile is opting out of caching due to an AST plugin that does not provide a caching strategy: `' + buildInfo.name + '`.');
-        cacheKeys.push((new Date()).getTime() + '|' + Math.random());
-      }
-    });
-
-    return {
-      plugins,
-      cacheKeys
-    };
+  build(options, cacheKey) {
```

Comment:
We need to deal with `cacheKey` being undefined below. Specifically if `ember-cli-htmlbars-inline-precompile` is being used by an older `broccoli-babel-transpiler` `cacheKey` would not have been passed in.